src/EEAlgorithm.jl

```diff
@@ -294,13 +655,20 @@ entry point to this jet reconstruction.
 - `clusterseq`: The resulting `ClusterSequence` object representing the
   reconstructed jets.
 """
-function _ee_genkt_algorithm!(particles::AbstractVector{EEJet};
-                              algorithm::JetAlgorithm.Algorithm, p::Real, R::Real = 4.0,
-                              recombine = addjets)
+function _ee_genkt_algorithm(; particles::AbstractVector{EEJet},
```

Comment:
```suggestion
function _ee_genkt_algorithm!(particles::AbstractVector{EEJet};
```

I think there was a mistake in resolving merge conflict